includes/ApplHandlingByStr.php

```diff
@@ -41,6 +82,44 @@ function showStatus( $stat_str )
 }
 
 
+function needAcceptor($status)
+{
+	$str = explode ( ';', $status );
+	for ( $i =  1;  $i < count ( $str )-1 ; $i++ )
+	{
+		$st = explode ( ',', $str[$i] )[1];
+		
+		if( $st == 'P' )
+		{
+			return false;
+		}
+	}
+	
+	if( explode(',', $str[ count($str)-1 ] )[1] == 'P' )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->